CONTRIBUTING.md

```diff
@@ -0,0 +1,158 @@
+# Contribute to WarpX
+
+We welcome new contributors! Here is how to participate to the WarpX 
+development.
+
+## Git workflow
+
+The WarpX project uses [git](https://git-scm.com) for version control. If you 
+are new to git, you can follow one of these tutorials:
+- [Learn git with bitbucket](https://www.atlassian.com/git/tutorials/learn-git-with-bitbucket-cloud)
+- [git - the simple guide](http://rogerdudler.github.io/git-guide/)
+
+### Create your branch from the `dev` branch
+
+Follow instructions from the 
+[WarpX documentation](https://ecp-warpx.github.io/doc_versions/dev/building/building.html) 
+to install the code. Make sure you are on WarpX `dev` branch with
+```
+git checkout dev
+```
+in the WarpX directory.
+
+Create a branch `<branch_name>` (the branch name should reflect the piece 
+of code you want to add, like `fix_spectral_solver`) with
+```
+git checkout -b <branch_name>
+```
+and do the coding you want. It is probably a good time to look at the 
+[AMReX documentation](https://amrex-codes.github.io/amrex/docs_html/). 
+Add the files you work on to the git staging area with 
+```
+git add file_I_created and_file_I_modified
```

Comment:
To follow the same convention as above, maybe this should be
```
git add <file_I_created> <and_file_I_modified>
```

Reply:
Yes, this is more consistent